lib/rucio/client/replicaclient.py

```diff
@@ -29,14 +29,34 @@ class ReplicaClient(BaseClient):
 
     REPLICAS_BASEURL = 'replicas'
 
-    def declare_bad_file_replicas(self, pfns, reason):
+    def quarantine_replicas(self, replicas, rse=None, rse_id=None):
+        """
+        Add quaratined replicas for RSE.
+
+        :param replicas: List of replica infos: {'scope': <scope> (optional), 'name': <name> (optional), 'path':<path> (required)}.
+        :param rse: RSE name.
+        :param rse_id: RSE id. Either RSE name or RSE id must be specified, but not both
+        """
+
+        if (rse is None) == (rse_id is None):
+            raise ValueError("Either RSE name or RSE id must be specified, but not both")
+
+        data = {'rse': rse, 'rse_id': rse_id, 'replicas': replicas}
+        url = build_url(self.host, path='/'.join([self.REPLICAS_BASEURL, 'quarantine']))
```

Comment:
Am I missing something, or is the `/replicas/quarantine` endpoint non-existent? I can't find it in [lib/rucio/web/rest/flaskapi/v1/replicas.py](https://github.com/rucio/rucio/pull/5773/files#diff-97d28c48bede938144ea7abff7f00e9ee4dc56f3249cb028395a198af731efbe)